mmcv/transform/loading.py

```diff
@@ -0,0 +1,241 @@
+        to_float32 (bool): Whether to convert the loaded image to a float32
+            numpy array. If set to False, the loaded image is an uint8 array.
+            Defaults to False.
+        color_type (str): The flag argument for :func:`mmcv.imfrombytes`.
+            Defaults to 'color'.
+        imdecode_backend (str): The image decoding backend type. The backend
+            argument for :func:`mmcv.imfrombytes`.
+            See :fun:`mmcv.imfrombytes` for details.
+            Defaults to 'cv2'.
+        file_client_args (dict): Arguments to instantiate a FileClient.
+            See :class:`mmcv.fileio.FileClient` for details.
+            Defaults to ``dict(backend='disk')``.
+    """
+
+    def __init__(
+        self,
+        to_float32: bool = False,
+        color_type: str = 'color',
+        imdecode_backend: str = 'cv2',
+        file_client_args: dict = dict(backend='disk')
+    ) -> None:
+        self.to_float32 = to_float32
+        self.color_type = color_type
+        self.imdecode_backend = imdecode_backend
+        self.file_client_args = file_client_args.copy()
+        self.file_client = None
+
+    def transform(self, results: dict) -> dict:
+        """Functions to load image.
+
+        Args:
+            results (dict): Result dict from :obj:`mmdet.CustomDataset`.
+
+        Returns:
+            dict: The dict contains loaded image and meta information.
+        """
+
+        if self.file_client is None:
+            self.file_client = mmcv.FileClient(**self.file_client_args)
+
+        filename = results['img_path']
+        img_bytes = self.file_client.get(filename)
+        img = mmcv.imfrombytes(
+            img_bytes, flag=self.color_type, backend=self.imdecode_backend)
+        if self.to_float32:
+            img = img.astype(np.float32)
+
+        results['img'] = img
+        height, width = img.shape[:2]
+        results['height'] = height
+        results['width'] = width
+        results['ori_height'] = height
+        results['ori_width'] = width
+        return results
+
+    def __repr__(self):
+        repr_str = (f'{self.__class__.__name__}('
+                    f'to_float32={self.to_float32}, '
+                    f"color_type='{self.color_type}', "
+                    f"imdecode_backend='{self.imdecode_backend}', "
+                    f'file_client_args={self.file_client_args})')
+        return repr_str
+
+
+class LoadAnnotation(BaseTransform):
+    """ Load and process the `instances` and `seg_map` annotation that dataset
+    provides.
+
+    The annotation format as follow:
+    {
+        'instances':
+        [
+            {
+            # List of 4 numbers representing the bounding box of the
+            # instance, in (x1, y1, x2, y2) order.
+            'bbox': [x1, y1, x2, y2],
+
+            # Label of image classification.
+            'bbox_label': 1,
+
+            # Used in key point detection.
+            # In the format of [x1, y1, v1,…, xn, yn, vn]. v[i] means the
```

Comment:
I think we should tell the user that this function only supports  loading keypoints of three values.